<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_linkControl">
       <attribute name="title">
        <string>连接控制</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="connectToSever">
          <property name="text">
           <string>连接服务端</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_15">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string> 本地控制</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="Line" name="line_14">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="pushButton_readKeys">
               <property name="text">
                <string>读取按键状态</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_key1">
               <property name="styleSheet">
                <string notr="true">border-style:hidden;</string>
               </property>
               <property name="text">
                <string>按钮1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_key2">
               <property name="styleSheet">
                <string notr="true">border-style:hidden;</string>
               </property>
               <property name="text">
                <string>按钮2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_key3">
               <property name="styleSheet">
                <string notr="true">border-style:hidden;</string>
               </property>
               <property name="text">
                <string>按钮3</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_11">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QPushButton" name="pushButton_directControl">
               <property name="text">
                <string>直接控制灯</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_led1">
               <property name="styleSheet">
                <string notr="true">border-style:hidden;</string>
               </property>
               <property name="text">
                <string>灯1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_led2">
               <property name="styleSheet">
                <string notr="true">border-style:hidden;</string>
               </property>
               <property name="text">
                <string>灯2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_led3">
               <property name="styleSheet">
                <string notr="true">border-style:hidden;</string>
               </property>
               <property name="text">
                <string>灯3</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_12">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QPushButton" name="pushButton_recLed">
               <property name="text">
                <string>记录灯动作</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_recOperate">
               <property name="text">
                <string>记录操作记录</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_13">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QPushButton" name="pushButton_viewLedOp">
               <property name="text">
                <string>查看灯动作</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_viewOp">
               <property name="text">
                <string>查看操作记录</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_14">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>输出：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_console_2"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
       <zorder>connectToSever</zorder>
       <zorder>textEdit_console_2</zorder>
       <zorder>label_9</zorder>
       <zorder>line_7</zorder>
       <zorder>verticalSpacer</zorder>
       <zorder>groupBox</zorder>
       <zorder>line_15</zorder>
      </widget>
      <widget class="QWidget" name="tab_linkConfig">
       <attribute name="title">
        <string>连接配置</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>网络设置</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,1">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>地址:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_ca">
            <property name="placeholderText">
             <string>服务器地址</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>端口:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_cp">
            <property name="placeholderText">
             <string>端口</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,2,0,1">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>用户:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_ca_2">
            <property name="placeholderText">
             <string>用户名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>密码:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_cp_2">
            <property name="placeholderText">
             <string>密码</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>通信测试</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTextEdit" name="textEdit_cs">
            <property name="placeholderText">
             <string>请输入待发送数据</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="c_send">
            <property name="text">
             <string>发送目标数据</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <widget class="QTextEdit" name="textEdit_ca">
          <property name="placeholderText">
           <string>接受数据如下</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>输出：</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QTextEdit" name="textEdit_console"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
